### DISCUSSION TITLE
username keeps appending a 4 randnum suffix

### DISCUSSION THREAD

**Discussion by HelloMukama** [Radi85/Comment#195] [Q&A]
I embedded this app into my project but then the username is something like this `johndoe-fdru`
I had originally configured my `Profile` model to run the profile detail link as a slug of the username
Meaning originally i had `localhost/johndoe`
Now, im getting `localhost/johndoe-fdru`

How can i solve this yet i even deleted the `UserProfile` model that ships with this repo and in the `settings.py` explicitly defined the following as per the docs
```
PROFILE_APP_NAME = 'profiles'
PROFILE_MODEL_NAME = 'Profile' # letter case insensitive
```

**Comment by HelloMukama** [comment]
I saw the regex after the pk on the app's view profile url.
My qn is this is okay but why does the behaviour remain even after i delete the profiles app?

**Reply by abhiabhi94** [reply]
The profile application is not shipped with the app. The profile app is just an example application, used for testing.  You don't need to use it inside your project.

**Reply by HelloMukama** [reply]
its exactly what i did. i removed it but now the issue im having is the issue of those detail urls.
`johndoe-xxxx`
How do i solve this???
My personal `Profile` model implements the `slug` field meaning my users just go to a `localhost/username` but ever since i integrated the comments app, im getting this `localhost/username-xxxx` not `localhost/username`
